### ISSUE TITLE
[Error] [Solution] UnicodeEncodeError: 'charmap' codec can't encode character

### ISSUE THREAD

**Issue by 0uros** [daydiff/yt-playlist-export#2]
Hey there

First of all, thanks for creating this tool, it has been incredibly helpful! I used it successfully to export liked and watch later playlist (some with +1000 videos) from four different YouTube Channels. I haven't find a quick method to made a backup from these videos before, I'm really happy.

However, I encountered an issue when exporting playlists that contain special characters or emojis in the video title. The script throws the following error:

`UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f165' in position 160: character maps to <undefined>`

The issue occurs because the script does not specify the encoding when writing the CSV file. On Windows, the default encoding does not support certain characters (cp1252)

To fix this issue, I added `encoding="utf-8"` when opening the file in print_csv. Here's the updated code:

```python
def print_csv(info: any):
    global output
    if output is None:
        csv_file = sys.stdout
    else:
        csv_file = open(output, mode='w', newline='', encoding='utf-8')
    writer = csv.DictWriter(csv_file, fieldnames=header)
    writer.writeheader()
    for row in convert_to_csv(info):
        writer.writerow(row)
```

Additionally, I’d like to suggest adding documentation about the supported browsers and how to configure them. The tool automatically picked Firefox for me, which worked perfectly, but I normally use Brave and wasn’t sure if I needed to configure anything. Including some notes about this in the README would be helpful for other users.

Thank you again for your great work! Let me know if you'd like me to open a pull request with this fix.

**Comment by daydiff** [comment]
Hi @0uros,

thank you for finding the bug in the code and for your nice words. I would greatly appreciate if you open a pull request with the fix. If not, I'll do the fix myself this weekend.

**Comment by 0uros** [comment]
Hi daydiff,

I wanted to let you know that I’ve created a pull request to address the UnicodeEncodeError issue

Thanks again for this tool!

**Comment by daydiff** [comment]
Than you @0uros, your PR #3 is merged and new version `0.0.5` of the tool is published.
